index.html

```diff
@@ -71,6 +74,290 @@ <h2 class="intro-heading">I'am a Software Developer</h2>
+          <h3 class="project-title">Profesional Art Printing Data</h3>
+          <p class="project-details">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <ul class="project-technology">
+            <li>
+              <a href="https://html.com/" class="project-btn project-btn-1">
+                HTML</a
+              >
+            </li>
+            <li>
+              <a
+                href="https://getbootstrap.com/"
+                class="project-btn project-btn-2"
+                >Bootstrap</a
+              >
+            </li>
+            <li>
+              <a
+                href="https://www.ruby-lang.org/en/"
+                class="project-btn project-btn-3"
+                >Ruby</a
+              >
+            </li>
+          </ul>
+          <a href="#" class="see-project-btn">See project</a>
+        </div>
+      </section>
+      <section class="about-me-section">
+        <div class="about-me-intro">
+          <h2 class="about-me-title">About me</h2>
+          <p class="about-me-details">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hesitate to contact me.
+          </p>
+          <a href="#" class="btn-resume">Get my resume</a>
+        </div>
+        <hr id="skills-separator" />
+        <div class="skills-div-container">
+          <div class="skill-div-bock">
+            <h3 class="skill-title" id="skill-title-1">
+              <img
+                src="./images/rectanle.png"
+                alt="rectangle icon"
+                id="rectangle-icon-1"
+              />Language
+            </h3>
+            <ul class="skills-list">
+              <li>JavaSript</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </ul>
+            <h3 class="skill-title" id="skill-title-2">
+              <img
+                src="./images/rectanle.png"
+                alt="circle icon"
+                id="rectangle-icon-2"
+              />Framework
+            </h3>
+            <ul class="skills-list">
+              <li>React.js</li>
+              <li>Ruby on Rails</li>
+              <li>RSpec</li>
+              <li>Capybara</li>
+            </ul>
+
+            <h3 class="skill-title" id="skill-title-3">
+              <img
+                src="./images/circle.png"
+                alt="circle icon"
+                id="circle-icon"
+              />Skills
+            </h3>
+            <ul class="skills-list">
+              <li>Database Management</li>
+              <li>version Control</li>
+              <li>CLI</li>
+              <li>API Design</li>
+              <li>Web Development</li>
+            </ul>
+          </div>

```

Comment:
As per the project requirements, all the technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul`, each of them in a `li`.
- You have a `div` with class `skill-div-bock`. Let's make this a `ul`.
- Each of the list that is inside the `div`, lets wrap it an `li`.  Something like this. ⬇️
```
<li>
    <h3 class="skill-title" id="skill-title-1">
    <img
            src="./images/rectanle.png"
            alt="rectangle icon"
            id="rectangle-icon-1"
            />Language
            </h3>
            <ul class="skills-list">
              <li>JavaSript</li>
              <li>Ruby</li>
              <li>HTML</li>
              <li>CSS</li>
            </ul>
</li>
```